### ISSUE TITLE
Section and Chapters Folding

### ISSUE THREAD

**Issue by iamunhoz** [area/language-latex#117]
Hi people, I've looked for this in many packages but I couldn't find anywhere, so I decided to ask you here.

The thing I'm missing when I use Atom for Tex files is the folding function, which was present in notepad++ when I am on windows machines and on linux I can find it in Scite, but Atom is way too more elegant and organized than the others, so I prefer to stick to it.

Is this feasible? Thanks in advance.

**Comment by dmorgorg** [comment]
Hover next to the line number where you want to fold and a V or > appears to fold or unfold.

**Comment by clsmt** [comment]
What do you mean? Typing V or > doesn't fold sections in my .tex file.

**Comment by Aerijo** [comment]
Atom currently supports indentation folding. This means that folding will hide every line that starts one or more spaces past the beginning of the line you are trying to fold.

A solution would be to have section headings flush with the left margin, subsections indented by a single space, subsubsections by two spaces, etc., while doing the same thing with the text. This should not influence your output. Eg.

\section{S1}
 This text will be folded if \section{S1} is.
 \subsection{SS1}
  This is text for subsection1. Note that you can fold this text by clicking the arrow head on the gutter next to the line for \subsection{SS1}, or the one on the line for \section{S1} (in Atom; not here).

This text will not be folded, because it starts in the same column as the section command.

**Comment by Aerijo** [comment]
You can copy and paste that text into a document in Atom. Moving your mouse over the line numbers on the left hand side you will see the 'V' and '>' shapes appear next to the numbers (you don't type these). Clicking these activates/deactivates folding.

**Comment by Aerijo** [comment]
@iamunhoz As this issue is for Atom in general, and not this package, please close this issue. I explain it a little more in #132 (even though it's quite unrelated, the conversation went in the direction of folding). 

TL;DR Atom only does indentation folding.

**Comment by Aerijo** [comment]
I've implemented section folding as part of my word count package. It'll probably move to it's own package eventually, or tree-sitter will take care of it, but the issue can be closed here now.

https://atom.io/packages/latex-wordcount
